Doc/library/asyncio-task.rst

```diff
@@ -540,6 +540,14 @@ Task functions
 
    .. deprecated:: 3.4.4
 
+.. function:: wrap_future(future, \*, loop=None)
+
+   Wrap a :class:`concurrent.futures.Future` object in a
+   :class:`Future` object.
+
+   When the original :class:`concurrent.futures.Future` is completed
```

Comment:
Alright. Perhaps just remove this sentence then. 
@adisbladis Could you remove the sentence? Thanks.